tests/track/loader_test.py

```diff
@@ -1257,21 +1257,21 @@ def test_sets_absolute_path(self, path_exists):
 
 class TrackFilterTests(TestCase):
```

Comment:
Can you please change the test case method names? They sometimes reference the term "included" which is outdated now.